releases/v1.4.0-beta.toml

```diff
@@ -0,0 +1,72 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.3.0"
+
+pre_release = true
+
+preface = """\
+The fifth major release of containerd includes a mix of new features and
+expanded support, such as support for CGroups v2, expanded SELinux support,
+support for Windows on Kubernetes through CRI, and support for snapshotters
+based on shared remote storage. All significant bug and stability fixes included
```

Comment:
and support for stargz snapshotter 🙏

Reply:
Well see, not sure about adding in the first sentence since I would prefer to highlight the generic capability. Added mention under snapshotters and cri section though.